test_util/dcos_api_session.py

```diff
@@ -67,8 +48,6 @@ def api_request(self, method, path_extension, *, scheme=None, host=None, query=N
         node argument. node must be a host in self.master or self.all_slaves. If given,
         the request will be made to the Admin Router endpoint for that node type
         """
-        assert self.masters
-        assert self.all_slaves
```

Comment:
This was removed because it was too generic of an assert. What you recommend should work well, so I will add that.